docs/ENVIRONMENT.md

```
@@ -53,7 +53,7 @@ server.ts
 Each hook command in `.claude/settings.json` sets its variables inline:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)